scripts/install.tpl.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+olm_version=$OLM_V0_VERSION
+cert_mgr_version=$CERT_MGR_VERSION
+rukpak_version=$RUKPAK_VERSION
+operator_controller_version=$VERSION
+
+if [[ -z "$olm_version" || -z "$cert_mgr_version" || -z "$rukpak_version" || -z "$operator_controller_version" ]]; then
+    err="Error: Missing component version(s) for: "
+    if [[ -z "$olm_version" ]]; then
+        err+="operator-lifecycle-manager "
+    fi 
+    if [[ -z "$cert_mgr_version" ]]; then
+        err+="cert-manager "
+    fi 
+    if [[ -z "$rukpak_version" ]]; then
+        err+="rukpak "
```

Comment:
Nit, extra spaces.
```suggestion
        err+="operator-lifecycle-manager"
    fi 
    if [[ -z "$cert_mgr_version" ]]; then
        err+="cert-manager"
    fi 
    if [[ -z "$rukpak_version" ]]; then
        err+="rukpak"
```

Reply:
They were intentional, it's a lazy way to format the final error string when component versions are unset.